debian/control

```diff
@@ -24,6 +25,6 @@ Provides:
  grml-debian-keyring,
 Breaks:
  grml-debian-keyring (<< 2024.12.08),
-Description: GnuPG keys used by the Grml project
+Description: OpenPGP archive certificates off the grml.org repository
```

Comment:
```suggestion
Description: OpenPGP archive certificates of the grml.org repository
```

Reply:
Ah, thanks, and I notice that I botched the rebase, as the description had been updated during the package renames, I've adapted it now based on that one.